### PR TITLE
Kevin/feat:add landing page clone with adjustment

### PR DESCRIPTION
**Pull request by kevinw-binary**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — APPROVED** [review]

File: src/pages/landing/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        header: <Localize translate_text="Withdraw" />,
+        text: (
+            <Localize translate_text="Get your funds quickly and easily. We support a variety of withdrawal options." />
+        ),
+        icon: <img src={WithdrawIcon} alt="withdraw" width="32" height="32" />,
+    },
+]
+const Home = () => {
+    return (
+        <Layout is_ppc_redirect={true} is_ppc={true}>
```
Comment (sean-binary) [inline src/pages/landing/index.js:45]: Please add the `no_index` flag

**Review by sean-binary — CHANGES_REQUESTED** [review]

File: src/pages/landing/index.js (hunk 1 above)
Comment (sean-binary) [inline src/pages/landing/index.js:45]: I think you should include the `SEO` component here just with the `no_index` flag.

**Review by sean-binary — CHANGES_REQUESTED** [review]

**Review by sean-binary — DISMISSED** [review]

**Review by yashim-deriv — APPROVED** [review]

**Review by mustofa-binary — COMMENTED** [review]
is there any new texts addition to this PR?

**Comment by kevinw-binary** [comment]
> is there any new texts addition to this PR?

I don't think there's any, it only remove some text

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**